types/parquetjs/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for parquetjs 0.10
+// Project: https://github.com/ironSource/parquetjs
+// Definitions by: jon-fearer <https://github.com/jon-fearer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.7
+
+export { ParquetEnvelopeReader, ParquetReader } from './lib/reader';
+export { ParquetEnvelopeWriter, ParquetWriter, ParquetTransformer } from './lib/writer';
+export { ParquetSchema } from './lib/schema';
+export { shredder as ParquetShredder } from './lib/shred';
+
+declare namespace parquet { }
```

Comment:
What is the purpose of this line?

Reply:
Had that in there when first developing this for some reason, but doesn't seem to serve any purpose now. Removed